TLCGen.PluggedInItems/TLCGen.Generators.CCOL/SourceFiles/wtlleds.c

```diff
@@ -0,0 +1,232 @@
+/* WACHTTIJD-LANTAARN MET LEDS */
+/* =========================== */
+
+
+/* (C) Copyright 2003-2019 by A.C.M. van Grinsven. All rights reserved.	*/
```

Comment:
Is dit OK om op te nemen in de source van TLCGen? Dit staat onder een MIT licentie op Github.

Reply:
Nope, niet OK. Files had ik verwijderd uit project maar niet uit directory, bij sync toch mee gegaan.
Inmiddels verwijderd van Github. Dank!